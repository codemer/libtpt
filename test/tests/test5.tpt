This variable should be undefined: "${undefined}"
template_library: "${template_library}"
