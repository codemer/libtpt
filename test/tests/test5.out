This variable should be undefined: ""
template_library: "LibTPT"
